pullrequest.rst

```diff
@@ -119,10 +119,27 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you can add it to the list of changes to be committed (the "staging area")::
+
+   git add <file>
+
+After adding all files of changed, you can commit your staged changes::
+
+   git commit
+
+Inside the commit editor, commit messages title should startswith
+``bpo-NNNNN`` and the major changed of this commit, for example::
+
+   bpo-29624: Adds purge step and layout test after uploading files.
+
+You will saw hash tag number in ``git log``, e.g.
```

Comment:
I've rearrange the paragraph here. First introduce these command, and a note block to remind contributer won't need to add PR id to title. (I already make a mistake here......)